### ISSUE TITLE
`vue/no-literals-in-template` should skip `xxx-style` and `xxx-class`

### ISSUE THREAD

**Issue by ZacharyBear** [vuejs/eslint-plugin-vue#3071]
**What rule do you want to change?**
Some components accept a `xxx-style` or `xxx-class` attribute, it would be passed to its child components or elements. In this case, they should be allowed.
`vue/no-literals-in-template` didn't recognized these props.

**Does this change cause the rule to produce more or fewer warnings?**
Fewer

**How will the change be implemented? (New option, new default behavior, etc.)?**
As the property's type implements `ClassValue` or `StyleValue`, they should be skipped.

**Please provide some example code that this change will affect:**

```vue
  <ElScrollbar
    :view-class="[{ filled: props.fill }]"
  />
```

**What does the rule currently do for this code?**
It passed classes to `ElScrollbar`, then `ElScrollbar` will pass these classes to it's element `div.el-scrollbar__view`.

**What will the rule do after it's changed?**
It will skip prefixed class or style properties.

**Additional context**

`string`, `Record<string, any>` and some types is implemented `ClassValue`, it's hard to recognize what is a  attribute should be skipped.

**Comment by ZacharyBear** [comment]
How about add an `ignores` option? That developers can specify which props shouldn't trigger this rule check.

**Comment by waynzh** [comment]
> How about add an `ignores` option

PR welcome for that! 🙂
